tests/SapNwRfc.Tests/Internal/OutputMapperTests.cs

```diff
@@ -401,6 +401,64 @@ private sealed class TimeSpanModel
             public TimeSpan? NullableTimeSpanValue { get; set; }
         }
 
+        private delegate void GetBytesCallback(IntPtr dataHandle, string name, byte[] buffer, uint bufferLength, out RfcErrorInfo errorInfo);
+
+        [Theory]
+        [InlineData(new byte[] { 1, 2, 3 })]
+        public void Extract_ByteArray_ShouldMapFromByteArray(byte[] value)
+        {
+            // Arrange
+            RfcErrorInfo errorInfo;
+
+            _interopMock
+               .Setup(x => x.GetBytes(It.IsAny<IntPtr>(), It.IsAny<string>(), It.IsAny<byte[]>(), It.IsAny<uint>(), out errorInfo))
+               .Callback(new GetBytesCallback((IntPtr dataHandle, string name, byte[] buffer, uint bufferLength, out RfcErrorInfo ei) =>
+               {
+                   Array.Copy(value, buffer, value.Length);
+                   ei = default;
+               }));
+
+            // Act
+            BytesModel result = OutputMapper.Extract<BytesModel>(_interopMock.Object, DataHandle);
+
+            // Assert
+            _interopMock.Verify(
+                x => x.GetBytes(DataHandle, "BYTESVALUE", It.IsAny<byte[]>(), It.IsAny<uint>(), out errorInfo),
```

Comment:
Would be good to test if the correct length has been passed to `GetBytes` (3 instead of `It.IsAny<uint>()`).